bitmapist/__init__.py

```diff
@@ -226,6 +226,18 @@ def delete_temporary_bitop_keys(system='default'):
         cli.delete(*keys)
 
 
```

Comment:
- Shouldn't we inherit `BITOP_KEYS` from `threading.local`? Current implementation is not thread-safe. 
- Removal of non-exising keys doesn't raise any exceptions in Redis.
- If we sort keys by system we can remove them with one [DEL command](http://redis.io/commands/del).